test_infra/stacks/base_stack.py

```diff
@@ -127,6 +127,27 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs: str) -> None:
                 ),
             },
         )
+        athena_spark_exec_role = iam.Role(
+            self,
+            "aws-sdk-pandas-athena-spark-exec-role",
+            role_name="AthenaSparkExecutionRole",
+            assumed_by=iam.ServicePrincipal("athena.amazonaws.com"),
+            managed_policies=[
+                iam.ManagedPolicy.from_aws_managed_policy_name("AmazonS3FullAccess"),
+                iam.ManagedPolicy.from_aws_managed_policy_name("AWSGlueConsoleFullAccess"),
+                iam.ManagedPolicy.from_aws_managed_policy_name("AmazonAthenaFullAccess"),
+            ],
+            inline_policies={
+                "GetDataAccess": iam.PolicyDocument(
+                    statements=[
+                        iam.PolicyStatement(
+                            actions=["lakeformation:GetDataAccess"],
+                            resources=["*"],
+                        ),
+                    ]
+                ),
```

Comment:
Nitpick - I feel like these managed policies and LF GetDataAccess inline policy is specified 3-4 times in this base stack, might be worth putting it in a dict and passing it around to reduce LOC?

Reply:
Agreed